BasicGates/BasicGates.ipynb

```diff
@@ -30,6 +30,22 @@
     "In all tasks you need to implement the transformation exactly, without introducing any global phase. You can read more about the global phase of quantum states [here](../tutorials/Qubit/Qubit.ipynb#Relative-and-Global-Phase)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> Executing the below cell will change the way qubits are displayed in the tasks results, this is consistent with the notation which we use throughout the Katas."
```

Comment:
This a way better solution. I have added a `.iqsharp-config.json` file with the config to the root folders, this will load the settings without having to execute any cells. As a bonus this will directly also work for the Workbooks.